test/apps/uct_info/check_tls.sh.in

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+res=true;
+for run in ./uct_info_static ./uct_info; do
+    echo Check for $run;
+    output=`$run`;
+    for i in @uct_test_tls@; do
```

Comment:
maybe pass list of transports as parameter to this script, from azure yml file?
this way it will be more flexible and not depend on m4 processing

Reply:
> maybe pass list of transports as parameter to this script, from azure yml file?
> this way it will be more flexible and not depend on m4 processing

done. for now there is no extra pls due to docker limitations (no ptrace permissions), will be extended later